appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: build{build}
+
+
+environment:
+  GherkinVersion: "3.3.0"
```

Comment:
The next version is going to be 4.0.0, but we didn't know that until recently - we thought it was going to be 3.2.1 until we introduced a backwards-incompatible change.

We won't always know what the next version will be, but I guess it's best to assume it will be the last release +1 patch - then update minor or major if we know it will change.

Anyway - please change this to 4.0.0